django_select2/static/django_select2/django_select2.js

```diff
@@ -43,6 +43,18 @@
       var $element = $(element)
       if ($element.hasClass('django-select2-heavy')) {
         initHeavy($element, settings)
+        var $selectedOption = $element.find(':selected')
```

Comment:
I missed this comment when I first checked your comments. I removed those lines after having checked that nothing changes, then I tried to remember why they were there.

Originally I did two main changes to django_select2: I added the admin integration and a couple of heavy widgets that let the user automatically create instances on-the-fly (a sort of free+existing tags). Before submitting this PR, I separated the changes related to the two parts and put them in two different branches, and I guess those lines belong to the other branches and I accidentally left them here. Sorry!